libs/extensions/angular/localization/src/di-tokens.ts

```diff
@@ -32,4 +36,8 @@ interface KirbyExtensionsLocalizationToken {
    * @example 'Europe/Copenhagen'
    */
   timeZone: string;
+  /**
+   * @example { DKK: 'kr.', EUR: '€' }
+   */
+  currencyMappings?: CurrencyMappings;
```

Comment:
Consider adding a currencyMapping to one of the examples in amount.stories.ts to showcase it.